worker/pkg/benthos/transformers/generate_state.go

```diff
@@ -10,11 +11,15 @@ import (
 // +neosyncTransformerBuilder:generate:generateState
 
 func init() {
-	spec := bloblang.NewPluginSpec().Description("Randomly selects a US state and returns the two-character state code.")
-
+	spec := bloblang.NewPluginSpec().Param(bloblang.NewBoolParam("state_code"))
```

Comment:
this should be optional and have a default set to false. also needs a top level description and param level description